exercises/pangram/example.R

```diff
@@ -0,0 +1,14 @@
+library(magrittr)
+
+is_pangram <- function(input) {
+
+  input_letters <- gsub("[^A-Za-z]", "", input) %>% 
+                    tolower() %>% 
+                    strsplit("") %>% 
+                    unlist() %>% 
+                    unique() %>% 
+                    sort()
```

Comment:
One can omit empty `()` for functions that are part of a pipe, but in the interest of differentiating it them from other variables, we should keep them in the examples.